sdk/computefleet/azure-resourcemanager-computefleet/src/test/java/com/azure/resourcemanager/computefleet/ComputeFleetManagerTests.java

```diff
@@ -0,0 +1,283 @@
+        } else {
+            resourceManager.resourceGroups()
+                .define(resourceGroupName)
+                .withRegion(REGION)
+                .create();
+        }
+    }
+
+    @Override
+    protected void afterTest() {
+        if (!testEnv) {
+            resourceManager.resourceGroups().beginDeleteByName(resourceGroupName);
+        }
+    }
+
+    @Test
+    @LiveOnly
+    public void testCreateComputeFleet() {
+        Fleet fleet = null;
+        try {
+            String fleetName = "fleet" + randomPadding();
+            String vmName = "vm" + randomPadding();
+            String vnetName = "vnet" + randomPadding();
+            String loadBalancerName = "loadBalancer" + randomPadding();
+            String adminUser = "adminUser" + randomPadding();
+            String adminPwd = UUID.randomUUID().toString().replace("-", "@").substring(0, 13);
+            // @embedStart
+            Network network = networkManager.networks()
+                .define(vnetName)
+                .withRegion(REGION)
+                .withExistingResourceGroup(resourceGroupName)
+                .withAddressSpace("172.16.0.0/16")
+                .defineSubnet("default")
+                .withAddressPrefix("172.16.0.0/24")
+                .attach()
+                .create();
+
+            LoadBalancer loadBalancer = networkManager.loadBalancers()
+                .define(loadBalancerName)
+                .withRegion(REGION)
+                .withExistingResourceGroup(resourceGroupName)
+                .defineLoadBalancingRule(loadBalancerName + "-lbrule")
+                .withProtocol(TransportProtocol.TCP)
+                .fromExistingSubnet(network, "default")
+                .fromFrontendPort(80)
+                .toBackend(loadBalancerName + "-backend")
+                .toBackendPort(80)
+                .attach()
+                .withSku(LoadBalancerSkuType.STANDARD)
+                .create();
+
+            fleet = computeFleetManager.fleets()
+                .define(fleetName)
+                .withRegion(REGION)
+                .withExistingResourceGroup(resourceGroupName)
+                .withProperties(
+                    new FleetProperties()
+                        .withSpotPriorityProfile(
+                            new SpotPriorityProfile()
+                                .withMaintain(true)
+                                .withCapacity(1)
+                                .withEvictionPolicy(EvictionPolicy.DELETE)
+                                .withAllocationStrategy(SpotAllocationStrategy.LOWEST_PRICE)
+                        )
+                        .withVmSizesProfile(
+                            Arrays.asList(
+                                new VmSizeProfile().withName("Standard_D4s_v3"),
+                                new VmSizeProfile().withName("Standard_D2s_v5"),
+                                new VmSizeProfile().withName("Standard_E2s_v3")
```

Comment:
Why three VM sizes?

Reply:
Maintain is only allowed on Fleet which contains at least 3 VM sizes in properties.vmSizesProfile.

Reply:
We could unset `withMaintain(true)` in `SpotPriorityProfile`, and keep one VM size here. We just want basic functionalities.

Reply:
Fixed in the new version.